bridge/client/sdp.js

```diff
@@ -176,6 +182,7 @@ if (typeof(SDP) == "undefined")
             sdpObj.startTime = parseInt(sessionTime[1]);
             sdpObj.stopTime = parseInt(sessionTime[2]);
         }
+
```

Comment:
I don't know our policy for unrelated white space fixes. Perhaps it's safest to remove this.